src/test/integration/integration_tests.py

```diff
@@ -0,0 +1,276 @@
+import unittest
+import requests
+import argparse
+import json
+import sys
+import logging
+from datetime import date, datetime
+from urllib import parse
+from random import randint
+
+
+class integration_tests(unittest.TestCase):
+
+    # Set class variables related to configurations for tests
+    @classmethod
+    def set_configuration_variables(cls, configuration):
+        config_file_raw = open(configuration)
+        config_file_json = json.load(config_file_raw)
+
+        base_url = config_file_json["hostname"] + "v1/hr"
+        cls.departments_url = base_url + "/departments"
+        cls.positions_url = base_url + "/positions"
+        cls.locations_url = base_url + "/locations"
+
+        # This should be coming from a /businesscenters endpoint
+        cls.business_centers = ["AABC", "AMBC", "ASBC", "BEBC",
+                                "CCBO", "FOBC", "HSBC", "UABC"]
+
+        # Only student positions are currently supported.
+        cls.position_type = "student"
+
+        # Minimal number of locations to count for
+        cls.minimal_location_count = 200
+
+        client_id = config_file_json["client_id"]
+        client_secret = config_file_json["client_secret"]
+
+        access_token = cls.get_access_token(
+                config_file_json["token_api"],
+                client_id,
+                client_secret)
+
+        # Set headers and query parameters
+        cls.auth_header = {'Authorization': 'Bearer %s' % access_token}
```

Comment:
In your code, you used  both `'%s %s' % ('one', 'two')` and `'{} {}'.format('one', 'two')` two different style for the string formatting. I will recommend you stick with one of the style for the code consistency.

Reference: https://pyformat.info/

Reply:
Good catch.